ani.ter-markosyan/repo/Lesson_3/exercise_17/exercise_17.py

```diff
@@ -0,0 +1,22 @@
+#!usr/bin/evn python3
+
+number1 = int(input("Input the first number: "))
+number2 = int(input("Input second number: "))
+number3 = int(input("input third number: "))
+print("Input three different integers:", number1, number2, number3)
```

Comment:
RT: Use the user input in this line.

Hint: Use string concatenation to separate the user input and use it as an integer to your program.